api/envoy/config/filter/http/transformation/v2/transformation_filter.proto

```diff
@@ -23,6 +24,12 @@ message FilterTransformations {
   // Only RouteTransformations.RouteTransformation with matching stage will be
   // used with this filter.
   uint32 stage = 2 [ (validate.rules).uint32 = {lte : 10} ];
+
+  // Logs request/response sensitive information
+  // By default, this is false so no request or response sensitive information is logged.
+  // If set to true, the filter will log the request/response body and headers before 
+  // any transformation is applied.
```

Comment:
per my earlier comment, this will now also log after the transformation. please update the api to match

Reply:
Resolved here: https://github.com/solo-io/envoy-gloo/pull/240/commits/a767130c44692fb7a593df256b4b07ce0abb4c96